arangod/Graph/GraphManager.cpp

```diff
@@ -448,7 +442,7 @@ OperationResult GraphManager::storeGraph(Graph const& graph, bool waitForSync,
                                                  builder.slice(), options);
 
   if (!result.ok()) {
-    trx.finish(result.result);
+    std::ignore = trx.finish(result.result);
```

Comment:
I would be in favor or merging this failure block with the block below.
There should only be one unconditional call to finish with `result.result` as its argument.